.env

```diff
@@ -1,3 +1,13 @@
+REACT_APP_API_URI=
```

Comment:
Unless this has to do with the CI-pipeline, we are not using this. `REACT_APP_PROFILE_GRAPHQL` is used for specifying the URL to profile-api(graphql).

Reply:
Removed references to API_URI